jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml

```diff
@@ -318,6 +318,18 @@
                 export JENKINS_USE_SERVER_VERSION="y"
                 export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
                 export GINKGO_PARALLEL="y"
+                export ZONE="us-central1-b"
+                export E2E_OPT="--check_version_skew=false"
+        - 'gke-prod-smoke':  # kubernetes-e2e-gke-prod-parallel
```

Comment:
You're running this in the same project? Won't that cause the resource leak detection code to fail?